modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/security/IgniteSessionRepository.java

```diff
@@ -117,12 +142,33 @@ public IgniteSessionRepository setDefaultMaxInactiveInterval(int dfltMaxInactive
     }
 
     /** {@inheritDoc} */
-    @Override public void delete(String id) {
+    @Override public void delete(String sesId) {
         try {
-            cache().remove(id);
+            txMgr.doInTransaction(() -> {
+                Set<String> accEmails = sesToAccIdx.get(sesId);
+
+                sesToAccIdx.delete(sesId);
+
+                accEmails.forEach(sesToAccIdx::delete);
```

Comment:
Why you delete from same index `sesToAccIdx` twice?

Reply:
Good catch! That was a typo.